libdeno/binding.cc

```diff
@@ -408,6 +425,44 @@ v8::MaybeLocal<v8::Module> ResolveCallback(v8::Local<v8::Context> context,
   v8::EscapableHandleScope handle_scope(isolate);
   v8::Context::Scope context_scope(context);
 
+  v8::String::Utf8Value specifier_utf8val(isolate, specifier);
+  const char* specifier_cstr = ToCString(specifier_utf8val);
+
+  auto builtin_modules = d->GetBuiltinModules();
+  bool has_builtin = builtin_modules->Has(context, specifier).ToChecked();
+
+  // printf("has_builtin %s %d\n", specifier_cstr, has_builtin);
+  if (has_builtin) {
+    auto val = builtin_modules->Get(context, specifier).ToLocalChecked();
+    CHECK(val->IsObject());
+    auto obj = val->ToObject(isolate);
+
+    std::string src = "let globalEval = eval\nlet g = globalEval('this');\n";
```

Comment:
Color me somewhat surprised that creating an internal module must be done in such a "kludgy" way ... but then I don't see big objections.

Reply:
I've added a comment and a TODO to find a better way. 
```
    // In order to export obj as a module, we must iterate over its properties
    // and export them each individually.
    // TODO Find a better way to do this.
```

This was a last resort after trying several other ways of exporting the object...